onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/Transfer.java

```diff
@@ -54,6 +54,12 @@
   @CsvField(optional = true)
   private int minTransferTime = MISSING_VALUE;
 
+  @CsvField(optional = true)
+  private int minWheelchairTime = MISSING_VALUE;
+
+  @CsvField(optional = true)
+  private int wheelchairTransfer;
```

Comment:
Should the default be 0 here?

Reply:
I was about to respond that it would be 0 anyway as 0 is default for integers in Java, but then I read 
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html, which says "Relying on such default values, however, is generally considered bad programming style." ;) so I've fixed it.